### PR TITLE
Celeste 64: Add CHANGELOG.md

### PR DESCRIPTION
**Pull request by PoryGone**
## What is this fixing or adding?
Adds a changelog markdown file to the Celeste 64 World.

## How was this tested?
I read it, and tossed it into a markdown renderer.

### CODE REVIEW

File: worlds/celeste64/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Celeste 64 - Changelog
+
+
+## v1.1 - First Stable Release
+
+### Features
+
+- Goal is to collect a cetain amount of Strawberries and visit Badeline on her island
```
Comment (nicholassaylor) [inline worlds/celeste64/CHANGELOG.md:8]:
```suggestion
- Goal is to collect a certain amount of Strawberries and visit Badeline on her island
```

**Review by nicholassaylor — CHANGES_REQUESTED** [review]
One typo, otherwise good

**Review by nicholassaylor — APPROVED** [review]
LGTM

File: worlds/celeste64/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Celeste 64 - Changelog
+
+
+## v1.1 - First Stable Release
+
+### Features
```
Comment (Exempt-Medic) [inline worlds/celeste64/CHANGELOG.md:6]:
```suggestion
### Features:
```

File: worlds/celeste64/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Goal is to collect a certain amount of Strawberries and visit Badeline on her island
+- Locations included:
+	- Strawberries
+- Items included
```
Comment (Exempt-Medic) [inline worlds/celeste64/CHANGELOG.md:11]:
```suggestion
- Items included:
```

**Review by Exempt-Medic — CHANGES_REQUESTED** [review]
Minor formatting suggestions, did not do a close reading

**Review by Exempt-Medic — APPROVED** [review]

**Review by ThePhar — APPROVED** [review]

**Pull request merged**